CHANGELOG.md

```diff
@@ -3,10 +3,10 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 ### Added
-- Added base64 gem to gemspec (#218)[https://github.com/opensearch-project/opensearch-ruby/pull/218]
 ### Changed
 ### Deprecated
 ### Removed
+- Removed dependency on the base64 gem to avoid a warning with Ruby 3.3 ([#221](https://github.com/opensearch-project/opensearch-ruby/pull/221))
```

Comment:
Nitpicking, we don't have to explain the why in CHANGELOG, just the what. So I'd remove "to avoid a warning with Ruby 3.3"